cpp/tests/utilities/column_utilities.cu

```diff
@@ -290,6 +290,16 @@ std::vector<bitmask_type> bitmask_to_host(cudf::column_view const& c)
   }
 }
 
+template <typename T>
+static auto numeric_to_string_precise(T value)
+{
+  std::ostringstream o;
+
+  o << std::setprecision(std::numeric_limits<T>::max_digits10) << value;
+
+  return o.str();
+}
+
 struct column_view_printer {
   template <typename Element, typename std::enable_if_t<is_numeric<Element>()>* = nullptr>
   void operator()(cudf::column_view const& col, std::vector<std::string>& out)
```

Comment:
return the vector container.  (change at all places & usage).

Reply:
@karthikeyann I agree that'd be a better design, but it seems unrelated to this change and belongs in a separate PR.

Reply:
right.

Reply:
:+1: